### PR TITLE
[15.0][IMP][FWD] account_financial_report: extensibility: ml fields

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Forward port of https://github.com/OCA/account-financial-reporting/pull/951

cc https://github.com/ForgeFlow

### CODE REVIEW

**Review by DavidJForgeFlow — APPROVED** [review]
LGTM! Thanks Aaron

**Review by mariadforgeflow — APPROVED** [review]
Code review, LGTM!

File: account_financial_report/report/abstract_report_xlsx.py [hunk 1]
```diff
@@ -300,7 +300,9 @@ def write_line_from_dict(self, line_dict, report_data):
                     value or "",
                     report_data["formats"]["format_right"],
                 )
-        report_data["row_pos"] += 1
+            else:
+                self.write_non_standard_column(cell_type, col_pos, value)
+        self.row_pos += 1
```
Comment (pedrobaeza) [inline account_financial_report/report/abstract_report_xlsx.py]: Same question than in 14.0 PR.

Comment (AaronHForgeFlow) [reply account_financial_report/report/abstract_report_xlsx.py]: Tested in this version now :)

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

**Pull request merged**